src/services/AbstractDFG.jl

```diff
@@ -449,7 +493,7 @@ Return boolean whether a factor `label` is present in `<:AbstractDFG`.
 """
 function hasFactor(dfg::G, label::Symbol)::Bool where {G <: AbstractDFG}
 	@warn "hasFactor() deprecated, please use exists()"
-	return haskey(dfg.labelDict, label)
+	return exists(dfg, label)
```

Comment:
I don't use labeldict in LighDFG, seems like its replaced with exist everywhere.

Reply:
oh, good ot know thanks.  I have been using `haskey`